lstchain/scripts/tests/test_lstchain_scripts.py

```diff
@@ -213,6 +234,17 @@ def test_validity_tune_nsb(tune_nsb):
             assert line == '  "extra_noise_in_bright_pixels": 0.0'
 
 
+def test_validity_tune_nsb_waveform(tune_nsb_waveform):
+    output_lines = tune_nsb_waveform.stdout.splitlines()
+    for line in output_lines:
+        if '"nsb_tuning"' in line:
+            assert line == '  "nsb_tuning": true,'
+        if '"nsb_tuning_ratio"' in line:
+            assert line == '  "nsb_tuning_ratio": -1.0,'
```

Comment:
May be comment this result is meaningless because the input data do not allow a full test of the functionality (hence it is only a "formal" test that it does run)

Reply:
sure

Reply:
done